test/integrationTests/codeActionRename.integration.test.ts

```diff
@@ -19,8 +20,8 @@ suite(`Code Action Rename ${testAssetWorkspace.description}`, function () {
 
     suiteSetup(async function () {
         should();
-        await testAssetWorkspace.restore();
         await activateCSharpExtension();
+        await testAssetWorkspace.restore();
```

Comment:
I noticed this reordering of activation and workspace restore in the tests and was curious. What problem is this solving?

Reply:
Don't remember for sure. Its possible that this change is unnecessary since i think latter one where i changed tests to use real restore eventing instead of dummy one was root cause for issues with workspace restore instability during tests.